lib/k8s/client.rb

```diff
@@ -21,7 +21,8 @@
 
 module K8s
   # @param server [String] http/s URL
-  # @param options [Hash] @see Transport.new
+  # @param options [Hash]
+  # @param (see Transport#initialize)
```

Comment:
This syntax copies the params from the referenced method to this one. The link didn't work anyway.